helix-core/src/main/java/org/apache/helix/model/InstanceConfig.java

```diff
@@ -48,9 +48,13 @@ public class InstanceConfig extends HelixProperty {
    * Configurable characteristics of an instance
    */
   public enum InstanceConfigProperty {
-    HELIX_HOST, HELIX_PORT, HELIX_ZONE_ID, @Deprecated
+    HELIX_HOST,
+    HELIX_PORT,
+    HELIX_ZONE_ID,
+    @Deprecated
     HELIX_ENABLED,
     HELIX_ENABLED_TIMESTAMP,
```

Comment:
If we deprecate enable field, then we should deprecate enable timestamp and add a new entry for operation timestamp as a list.

Reply:
I think we should still keep HELIX_ENABLED_TIMESTAMP so re-enabling or disabling it when it is already in that state does not reset the delay window. Let's discuss this more offline.